app/home.tmpl

```diff
@@ -1,6 +1,7 @@
 {{ $home_container := where $ "ID" .Docker.CurrentContainerID | first }}
 {{ $show_footer := or (index $home_container.Labels "traefik-home.show-footer") "true" | parseBool }}
 {{ $show_status_dot := or (index $home_container.Labels "traefik-home.show-status-dot") "true" | parseBool }}
+{{ $sort_order := or (index $home_container.Labels "traefik-home.sort-by") "default" }}
```

Comment:
I'd like to keep the naming between the option variable and the label consistent.
Can you rename `$sort_order` to `$sort_by`?